helios_auth/templates/index.html

```diff
@@ -1,14 +1,14 @@
 {% extends TEMPLATE_BASE %}
-
+{% load i18n %}
 {% block content %}
-<h1>Authentication</h1>
+<h1>{% trans 'Authentication' %}</h1>
 
 {% if user %}
 <p style="font-size: 1.4em;">
-    You are currently logged in as<br /><b>{{user.user_id}}</b> via <b>{{user.user_type}}</b>.
+    {% blocktrans %}You are currently logged in as<br /><b>{{user.user_id}}</b> via <b>{{user.user_type}}</b>.{% endblocktrans %}
```

Comment:
Po [dokumentaciji](https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#blocktrans-template-tag) sodeč to ne bo delovalo - treba bo naredit nekako tako:
```django
{% blocktrans with user_id=user.user_id user_type=user.user_type %}
You are currently logged in as<br /><b>{{user_id}}</b> via <b>{{user_type}}</b>.
{% endblocktrans %}
```